[Rider 2024.3.5] NoClassDefFoundError: com/intellij/sh/ShLanguage

### Environment
- IDE: Rider 2024.3.5
- Plugin: Mise 4.2.0
- Mise: mise@MiseVersion(major=2025, minor=2, patch=3)
- OS: Windows 10 10.0
- Java: 21.0.5 (JetBrains s.r.o.)

### Description
I just installed the mise plugin and I was trying to test a simple task. 
Contents of mise.toml:
```
[tasks.script-test]
description = 'Test'
run = 'pwsh -c "Write-Host `"Hello, world!`""'
```

### Stacktrace
```
java.lang.NoClassDefFoundError: com/intellij/sh/ShLanguage
	at com.github.l34130.mise.core.lang.injection.MiseTomlTaskInjectionContributor.getInjection(MiseTomlTaskInjectionContributor.kt:13)
	at com.intellij.psi.impl.source.tree.injected.LanguageInjectionPerformerAdapter.getLanguagesToInject(LanguageInjectionPerformerAdapter.java:22)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.processInPlaceInjectorsFor(InjectedLanguageManagerImpl.java:498)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.probeElementsUpInner(InjectedLanguageUtilBase.java:237)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.lambda$probeElementsUp$0(InjectedLanguageUtilBase.java:217)
	at com.intellij.openapi.application.impl.ReadActionCacheImpl$allowInWriteAction$1.invoke(ReadActionCacheImpl.kt:41)
	at com.intellij.openapi.application.impl.ReadActionCacheImpl$allowInWriteAction$1.invoke(ReadActionCacheImpl.kt:41)
	at com.intellij.openapi.application.impl.ReadActionCacheImpl.allowInWriteAction(ReadActionCacheImpl.kt:29)
	at com.intellij.openapi.application.impl.ReadActionCacheImpl.allowInWriteAction(ReadActionCacheImpl.kt:41)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.probeElementsUp(InjectedLanguageUtilBase.java:216)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.enumerate(InjectedLanguageUtilBase.java:159)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.findInside(InjectedLanguageUtilBase.java:397)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.tryOffset(InjectedLanguageUtilBase.java:376)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.findInjectedElementNoCommit(InjectedLanguageUtilBase.java:357)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.findInjectedElementAt(InjectedLanguageManagerImpl.java:355)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtilBase.findInjectedPsiNoCommit(InjectedLanguageUtilBase.java:183)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getRegisteredIntentionActions(ShowIntentionsPass.java:362)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getActionsToShow(ShowIntentionsPass.java:350)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.doCollectInformation(ShowIntentionsPass.java:251)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:67)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:427)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceKt.use(trace.kt:30)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$3(PassExecutorService.java:423)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.tryRunReadAction(AnyThreadWriteThreadingSupport.kt:351)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:972)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$4(PassExecutorService.java:413)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$14(CoreProgressManager.java:674)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:749)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:673)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:79)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:412)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:388)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.cacheFileTypesInside(FileTypeManagerImpl.java:822)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$1(PassExecutorService.java:388)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.executeByImpatientReader(AnyThreadWriteThreadingSupport.kt:544)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:176)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:386)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:266)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
Caused by: java.lang.ClassNotFoundException: com.intellij.sh.ShLanguage PluginClassLoader(plugin=PluginDescriptor(name=Mise, id=com.github.l34130.mise, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\Rider2024.3\plugins\mise, version=4.2.0, package=null, isBundled=false), packagePrefix=null, state=active, parents=PluginDescriptor(name=Toml, id=org.toml.lang, moduleName=intellij.toml.json, descriptorPath=intellij.toml.json.xml, path=~\AppData\Roaming\JetBrains\Rider2024.3\plugins\toml, version=243.23654.183, package=org.toml.ide.json, isBundled=false), PluginDescriptor(name=Toml, id=org.toml.lang, moduleName=intellij.toml.grazie, descriptorPath=intellij.toml.grazie.xml, path=~\AppData\Roaming\JetBrains\Rider2024.3\plugins\toml, version=243.23654.183, package=com.intellij.toml.grazie, isBundled=false), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.impl, descriptorPath=intellij.platform.vcs.impl.xml, path=~\AppData\Local\Programs\Rider\lib, version=243.24978.27, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.log.impl, descriptorPath=intellij.platform.vcs.log.impl.xml, path=~\AppData\Local\Programs\Rider\lib, version=243.24978.27, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.dvcs.impl, descriptorPath=intellij.platform.vcs.dvcs.impl.xml, path=~\AppData\Local\Programs\Rider\lib, version=243.24978.27, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.rider, descriptorPath=intellij.rider.xml, path=~\AppData\Local\Programs\Rider\lib, version=243.24978.27, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.collaborationTools, descriptorPath=intellij.platform.collaborationTools.xml, path=~\AppData\Local\Programs\Rider\lib, version=243.24978.27, package=null, isBundled=true), PluginDescriptor(name=Toml, id=org.toml.lang, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\Rider2024.3\plugins\toml, version=243.23654.183, package=org.toml, isBundled=false), )
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:157)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 44 more

```

### Attachments

--- a/modules/core/src/main/kotlin/com/github/l34130/mise/core/lang/injection/MiseTomlTaskInjectionContributor.kt
+++ b/modules/core/src/main/kotlin/com/github/l34130/mise/core/lang/injection/MiseTomlTaskInjectionContributor.kt
@@ -10,6 +10,9 @@ import com.intellij.sh.ShLanguage
 class MiseTomlTaskInjectionContributor : LanguageInjectionContributor {
     override fun getInjection(context: PsiElement): Injection? {
         if (!MiseTomlPsiPatterns.inTaskRunString.accepts(context)) return null
-        return SimpleInjection(ShLanguage.INSTANCE, "", "", null)
+        return runCatching {
+            // Some IDEs doesn't have ShLanguage
+            SimpleInjection(ShLanguage.INSTANCE, "", "", null)
+        }.getOrNull()
     }
 }

